src/bigquery.ts

```diff
@@ -765,6 +792,46 @@ export class BigQuery extends common.Service {
     return BigQuery.timestamp(value);
   }
 
+  /**
+   * A BigQueryInt wraps 'INT64' values. Can be used to maintain precision.
+   *
+   * @method BigQuery#int
+   * @param {string|number|IntegerTypeCastValue} value The INT64 value to convert.
+   * @param {IntegerTypeCastOptions} typeCastOptions Configuration to convert
+   *     value. Must provide an `integerTypeCastFunction` to handle conversion.
+   *
+   * @example
+   * const {BigQuery} = require('@google-cloud/bigquery');
+   * const bigquery = new BigQuery();
+   *
+   * const largeIntegerValue = Number.MAX_SAFE_INTEGER + 1;
+   *
+   * customTypeCast = (value) => {
+   *    return value.split('');
+   * };
+   *
+   * const wrapIntegers = {
```

Comment:
```suggestion
   * const options = {
   *   integerTypeCastFunction: value => value.split(),
   * };
```

Reply:
This and the next are just some suggestions that could make it easier for the user to understand how the options all plug-in together.